lib/swagger-scanner.ts

```diff
@@ -18,7 +18,7 @@ export class SwaggerScanner {
         const path = metatype
           ? Reflect.getMetadata(MODULE_PATH, metatype)
           : undefined;
-        this.scanModuleRoutes(routes, path);
+        return this.scanModuleRoutes(routes, path);
```

Comment:
hi @xmlking , thank you for reporting,
it was a silly error, i missed the return here.
so it now should be fixed :smiley: